app/pages/_document.tsx

```diff
@@ -9,41 +9,46 @@ class MyDocument extends Document {
   render() {
     return (
       <Html>
-        <Head>
-          <meta name="viewport" content="width=device-width, initial-scale=1" />
-          <meta charSet="utf-8" />
-          <title>CleanBC Industrial Incentive Program</title>
-          <link rel="stylesheet" href="/static/bootstrap.min.css" />
-          <link rel="stylesheet" href="/static/base.css" />
-          <link
-            rel="apple-touch-icon"
-            href="/static/icons/bcid-apple-touch-icon.png"
-            sizes="180x180"
-          />
-          <link
-            rel="icon"
-            href="/static/icons/bcid-favicon-32x32.png"
-            sizes="32x32"
-            type="image/png"
-          />
-          <link
-            rel="icon"
-            href="/static/icons/bcid-favicon-16x16.png"
-            sizes="16x16"
-            type="image/png"
-          />
-          <link
-            rel="mask-icon"
-            href="/static/icons/bcid-apple-icon.svg"
-            color="#036"
-          />
-          <link rel="icon" href="/static/icons/bcid-favicon-32x32.png" />
-        </Head>
-        <body>
-          <Main />
-          <NextScript />
-          {CONFIG.ENABLE_ANALYTICS === 'true' && <Analytics />}
-        </body>
+        <html lang="en">
```

Comment:
Is there a reason for adding a second additional `<html>` tag here? There should be just [one per document](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html).

Reply:
I thought it was giving me a typescript error, but I think it was just my editor being really slow on validation